include/EventInserterPQ.h

```diff
@@ -0,0 +1,44 @@
+#ifndef EVENTINSERTERPQ_H
+#define EVENTINSERTERPQ_H
+
+#include <string>
+#include <vector>
+#include <map>
+#include <libpq-fe.h>
+//#include "Event.h"
+//#include "Root2Event.h"
+//#include "TupleReader.h"
+
+class EventInserterPQ{
+private:
+	//connector
+	PGconn* conn;
+	char* pghost = NULL;
+	char* pgport = NULL;
+	char* pgoptions = NULL;
+	char* pgtty = NULL;
+	std::string db_name;
+	char* login = NULL;
+	char* pwd = NULL;
+
+	//command execution
+	std::string table_name;
+	PGresult* res;
+	std::map<std::string, std::vector<std::string> > val_name;
+	
+	void CreateTable();
+	
+
+public:
+	EventInserterPQ() = default;
+	EventInserterPQ(std::string db_name_, std::string table_name_, std::map<std::string, std::vector<std::string> > val_name_);
+	~EventInserterPQ();
+
+	void EventInserterPQ::insert();
+
+};
+
+template<typename T>
+std::string to_string(std::vector<T> x);
```

Comment:
to_string is not a good name since it can potentially hide a function provided by the standard library. You never know what the standard library might provide in the future, so better to stay away from overloading functions having the same name as anything from the STL.